acceptancetests/assess_upgrade_lxd_profile.py

```diff
@@ -46,7 +46,7 @@ def deploy_bundle(client):
     client.wait_for(primary)
 
 def upgrade_charm(client):
-    client.upgrade_charm("lxd-profile", resvision='1')
+    client.upgrade_charm("lxd-profile", resvision='3')
```

Comment:
I thought this was a typo at first. We should probably rename this in the client

Reply:
I'll do that now